src/base/LockManagerBase.sol

```diff
@@ -41,9 +41,9 @@ abstract contract LockManagerBase is ILockManager {
 
     /// @notice Emitted when the plugin reports a proposal as ended
```

Comment:
@brickpop I know that I've reported this in another PR, but probably it's better to leave it here given that it's the PR that made the change.

👉 Change "as ended" to "as settled"